.github/workflows/release.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - run: |
-          git tag --force v2 HEAD
-          git push --force origin v2
+          git tag --force v3 HEAD
```

Comment:
All actions have migrate version number to integrate nodejs 20, I do the same.

Reply:
@ChristopheRivier To be honest, I am not sure what the common thing is here. Let me check with the team. I don't have any preference, but not sure what the impact of version bumping is here. How we would mark v2 as deprecated or even if we should. This is mostly due to my own lack of knowledge.

Reply:
@ChristopheRivier I chatted with the team. Since this is just a dependency upgrade, we would like to keep this at `v2` for the GHA version. Could you remove changes to this file?

Reply:
done